src/data_snack/wrap/data_frame.py

```diff
@@ -0,0 +1,39 @@
+from dataclasses import dataclass
+from typing import List, Text
+
+import pandas as pd
+
+from data_snack.wrap import EntityWrap
+from data_snack.wrap.exceptions import DataFrameMissingKeyColumn
+
+
+@dataclass
+class DataFrameWrap(EntityWrap):
```

Comment:
Do we need this class? Can't it be a feature of `EntityWrap` + you changed helper functions to this class' methods, so it works with simplified interface for wrapper only, but we have an alternative usage - non simplified. Maybe we can stay with functions and use them in `EntityWrap` as well to make this two "APIs" equal in terms of utility?